server/src/controllers/GameControllers/game.js

```diff
@@ -0,0 +1,112 @@
+import Snake from './snake';
+import SNAKE_CONSTANTS from './gameConstants';
+
+class Game {
+  constructor(playerOne, socketOne, io) {
```

Comment:
Use a single object for argument, it will help in remembering the argument name at the time of call, also you don't have to remember the arg position